content/200-concepts/100-components/02-prisma-client/260-client-extensions/110-client.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Add methods to Prisma Client'
+metaTitle: 'Prisma Client extensions: client component (Preview)'
+metaDescription: 'Extend the functionality of Prisma Client'
+tocDepth: 3
+---
+
+<TopBlock>
+
+<Admonition type="info">
+
+This page is in draft.
+
+</Admonition>
+
+You can use the `client` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component type to add top-level methods to Prisma Client. We introduced this feature in version 4.7.0.
+
+</TopBlock>
+
+## Enable the preview feature
+
+Before you create Prisma Client extensions, you must enable the `clientExtensions` feature flag in the `generator` block of your `schema.prisma` file, as follows:
+
+```prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["clientExtensions"]
+}
+```
+
+## Extend Prisma Client
+
+Use the `$extends` [client-level method](/reference/api-reference/prisma-client-reference#client-methods) to create an _extended client_. An extended client is a variant of the standard Prisma Client that is wrapped by one or more extensions. Use the `client` extension component to add top-level methods to Prisma Client.
+
+To add a top-level method to Prisma Client, use the following structure:
+
+```ts
+const xprisma = prisma.$extends({
+  client?: { ... }
+})
+```
+
+### Example
+
+The following example uses a `client` extension to start an [interactive transaction](/concepts/components/prisma-client/transactions#interactive-transactions) without callbacks.
```

Comment:
The example technically only shows how one would set up an extension that adds 2 methods, the implementation is completely missing. So the introduction here might give wrong hope.